testing/xfce4-sensors-plugin/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Maintainer:
```

Comment:
What if I want to contribute with one APKBUILD but don't want to take responsibility as maintainer?